resources/template/menu.php

```diff
@@ -0,0 +1,112 @@
+<?php
+namespace template;
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+use src\ProjectConfig;
+class Menu{
+    private $role;
+    private $uname;
+    private $current_file;
+    public $paths;
+
+    /*
+        * $role could have values: 'anon','user','admin'
+        * $file contains name of current file e.g 'index.php'
+    */
+    public function __construct($file,$role='anon',$name='Bookhive'){
+        $this->current_file = $file;
+        $this->role = $role;
+        $this->uname = $name;
+        $conf = new ProjectConfig();
+        $this->paths = $conf->config['paths'];
+        
+    }
+    public function render_header(){
+        ?>
+        <input type='checkbox' id='check'>
+            <!--header area start-->
+            <header>
+                <label for='check'>
+                    <i class='fas fa-bars' id='sidebar_btn'></i>
+                </label>
+                <div class='left_area'>
+                    <h3>Bookhive</h3>
+                </div>
+            </header>
+            <!--header area end-->
+            
```

Comment:
Try refactoring this, it doesn't look good to keep html like this in the classes. Try wrapping this inside echo / print statement instead.